website/content/en/highlights/2021-12-28-0-19-0-upgrade-guide.md

```diff
@@ -0,0 +1,51 @@
+---
+date: "2021-12-28"
+title: "0.19 Upgrade Guide"
+description: "An upgrade guide that addresses breaking changes in 0.19.0"
+authors: ["001wwang"]
+pr_numbers: []
+release: "0.19.0"
+hide_on_release_notes: false
+badges:
+  type: breaking change
+---
+
+Vector's 0.19.0 release includes **breaking changes**:
+
+1. [Removal of deprecated configuration fields for the Splunk HEC Logs sink: `host`](#splunk-hec-logs-sink-deprecated-fields)
+1. [Option name change in Splunk HEC sinks: `token` to `default_token`](#splunk-hec-token-to-default-token)
```

Comment:
This one technically isn't breaking since we added an alias. I think you can leave it out of this guide, which we like to have solely focused on breaking changes. I'm hoping to come up with a consistent method for documenting deprecations soon.